patternlab.info.yml

```diff
@@ -14,9 +14,6 @@ regions:
   sidebar_first: Sidebar first
   footer: Footer
 component-libraries:
```

Comment:
Noting here that we're going to move away from these sub-namespaces and only do full paths from now on.